flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskManagerComponentsStartupShutdownTest.java

```diff
@@ -202,8 +202,6 @@ protected void run() {
 			assertTrue(ioManager.isProperlyShutDown());
 			assertTrue(memManager.isShutdown());
 		} finally {
-			TestingUtils.stopActorsGracefully(Arrays.asList(jobManager, taskManager));
```

Comment:
Is this now happening automatically?

Reply:
If the test succeeds, they will have been shutdown (the previous asserts also assert (hehe) this.

It seems the previous Akka version didn't mind another shutdown attempt but 2.4 fails with an exception when you try to shutdown after already being shut down.